README.md

```diff
@@ -15,6 +15,117 @@ Supports Encrypted Media Extensions for playback of encrypted content in Video.j
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
+## Using
+
+By default, videojs-contrib-eme is not able to decrypt any audio/video. In order to
+decrypt audio/video, a user must pass in methods that are specific to a source and its
+combination of key system and codec. These are provided via videojs-contrib-eme's options.
+
+The bare minium requirement is to provide the `keySystems` property with an object
+containing at least one key system that you will be using and an implementation of its
+required methods.
+
+The `configurations` object contains audio and video sources mapped to their codecs.
+These are used to determine if the system supports that codec, and to create an
+appropriate `keySystemAccess` object. If left out, it is possible that the system will
+create a `keySystemAccess` object for the given key system, but will not be able to play
+the source due to the browser's inability to use that codec.
+
+### FairPlay
+
+For FairPlay, only `keySystems` is used from the options passed into videojs-contrib-eme.
+
+The required methods to provide are:
+* `getCertificate`
+* `getContentId`
+* `getConcatenatedInitData`
+* `getKey`
+
+Below is an example of videojs-contrib-eme options when only using FairPlay:
+
+```javascript
+{
+  keySystems: {
+    "com.apple.fps.1_0": {
+      getCertificate: (options, callback) => {
+        // request certificate
+        // if err, callback(err)
+        // if success, callback(null, certificate)
+      },
+      getContentId: (initData) => {
+        // return content ID
+      },
+      getConcatenatedInitData: (initData, certificate) => {
```

Comment:
Not sure this function is necessary. Was this one of the `ADAPT` functions from Apple's FP example?